docs/NewOperators.md

```diff
@@ -0,0 +1,24 @@
+## How to implement a new operator in Glow
+
+*Note: It's best to avoid implementing new Glow IR nodes and instructions. If support for a new operator is needed, it's best to first check to see if this operator can be implemented using an existing glow IR node or nodes. For example FCTransposed can be implemented using a regular FC node. If this is the case then the operator can be implemented by instantiating existing glow IR nodes in the loader or by simply creating a new node creation method in `Graph/Graph.cpp` which would create the sequence of operators that implements the new operator. Only in the event that this is not possible should new Glow IR nodes/instructions be created.*
+
+> Example PR: https://github.com/pytorch/glow/pull/2334
+
+### Implementation
+#### High level IR
+* Create a new Glow high level IR node in `ClassGen/NodeGen.cpp`. Run `ninja all` to generate the node.
+* Implement the `verify()` method for the new node in `Graph/Nodes.cpp`.
+* Implement a node creation method in `Graph/Graph.cpp`.
+* Implement logic to load model that contains the operator in `Importer/Caffe2ModelLoader.cpp` or `Importer/ONNXModelLoader.cpp` depending on which type of model the operator comes from. Add the operator to `Importer/CommonOperatorLoader.h` instead if the loading logic can be shared between Caffe2 and ONNX. Add as much validation logic as possible here in the loader for the operator because it's crucial to catch errors at this stage. Once the operator is loaded, it is assumed that Glow will be able to successfully run the operator so any issues must be caught here.
+#### Low level IR
+* Create a new Glow low level IR instruction in `ClassGen/InstrGen.cpp`.
+* If a custom translation from high level IR node to low level IR instruction is necessary then implement a new case in `IR/IRGen` otherwise just use `autoIRGen()` in the definition of the new IR instruction in `ClassGen/InstrGen.cpp`.
```

Comment:
when you specify autoIRGen() how would the mapping happen to the Node generated in the NodeGen. Can you add that?